editor/src/components/inspector/sections/layout-section/layout-system-subsection/layout-system-controls.tsx

```diff
@@ -273,6 +273,28 @@ export const PaddingControl = React.memo(() => {
     )
   }, [])
 
+  const selectedViews = useEditorState(
+    Substores.selectedViews,
+    selectedViewsSelector,
+    'PaddingControl selectedViews',
+  )
+
+  const onSplitChainedEvent: SplitChainedNumberInputEventHandler = React.useMemo(
+    () =>
+      longhandShorthandEventHandler(
+        'padding',
+        {
+          T: 'paddingTop',
+          R: 'paddingRight',
+          B: 'paddingBottom',
+          L: 'paddingLeft',
+        },
+        selectedViews[0],
```

Comment:
if you make longhandShorthandEventHandler take a selectedViewRef you can restore using the selectedViewsRef here! – since you removed the line `selectedViews={selectedViewsRef.current}` anyways